utils/bin/lvim

```diff
@@ -3,4 +3,25 @@
 export LUNARVIM_RUNTIME_DIR="${LUNARVIM_RUNTIME_DIR:-$HOME/.local/share/lunarvim}"
 export LUNARVIM_CONFIG_DIR="${LUNARVIM_CONFIG_DIR:-$HOME/.config/lvim}"
 
+for arg; do
+  shift
+  case $arg in
+    --update)
+      :
+      echo 'Updating LunarVim...'
+      old="$(pwd)"
+      cd "$LUNARVIM_RUNTIME_DIR/lvim"
+      target_branch=${1:"$(git rev-parse --abbrev-ref HEAD)"}
+      git stash --include-untracked -m "Changes before LunarVim update" && echo "Uncommitted changes saved to stash" || true
+      git fetch --force --depth=1 origin $target_branch:$target_branch
+      git checkout $target_branch
+      cd "$old"
+      nvim -u "$LUNARVIM_RUNTIME_DIR/lvim/init_update.lua" --noplugin -n -i NONE --headless
+
+      exit
+      ;;
+    *) set -- "$@" "$arg" ;;
```

Comment:
There shouldn't be a need for this when we can just call `:LvimUpdate` as long as the entry point is still `bootstrap.lua`, and we can add more handling for headless mode in the actual hook instead.

Reply:
Not really, `:LvimUpdate` is fragile and was largely the point of this PR. `:LvimUpdate` breaking is one of the most common complaints, and there is no benefit to having it run while an editor is open.

Doing updates _while the editor is running_ means that we can't guarantee the state of _anything_.

Users are also unable to simply switch branches, including to be able to switch to a PR to test a fix, for example.

`:LvimUpdate` not functioning or displaying errors is a complaint so common that I made something to simply link to, and the current fixes are _terrible_ and usually lead to a lot of errors:
https://github.com/LunarVim/LunarVim/discussions/1752

The approach to "fix things" or switch to rolling that I see most often mentioned is "just uninstall using this script and reinstall using this really long command":
```
bash ~/.local/share/lunarvim/lvim/utils/installer/uninstall.sh
LV_BRANCH=rolling bash <(curl -s https://raw.githubusercontent.com/lunarvim/lunarvim/rolling/utils/installer/install.sh)
```